feat(profiling): document how to work around broken fleet policies (#2539)
elastic/docs-content: release-notes/elastic-observability/known-issues.md

- [#220339](https://github.com/elastic/kibana/issues/220339)
- [#220342](https://github.com/elastic/kibana/issues/220342)

::::

::::{dropdown} Profiling Collector and Symbolizer endpoints are not configured after upgrading a cluster

Applies to: {{stack}} 9.x, 8.x

**Details**

> After upgrading a cluster, it may happen that Collector and Symbolizer endpoints are not configured, even though Universal Profiling has been set up in Kibana and the "Add data" instructions are shown on the Universal Profiling landing page.

Reviewer: ```suggestion
After upgrading a cluster, Collector and Symbolizer endpoints may not be configured even when Universal Profiling is enabled in Kibana and the "Add data" instructions appear on the Universal Profiling landing page.
```

**Workaround**

> 1. Run the following query and retrieve the `id` of the `elastic-universal-profiling-collector` and `elastic-universal-profiling-symbolizer` package policy

Reviewer: ```suggestion
1. Run the following query to retrieve the `id`s of the `elastic-universal-profiling-collector` and `elastic-universal-profiling-symbolizer` package policies:
```

    ```sh
    GET kbn:/api/fleet/package_policies
    ```

> 2. Execute the deletion.

Reviewer: ```suggestion
2. Delete the package policies:
```
